nodes/usb_cam_node.cpp

```diff
@@ -105,6 +106,11 @@ class UsbCamNode
     cam_.set_v4l_parameter("exposure_absolute", exposure_absolute);
   }
 
+  void gammaCallback(const std_msgs::Int32& msg)
+  {
+    cam_.set_v4l_parameter("gamma", msg.data);
```

Comment:
Don't you need a value validation?

Reply:
Valid range of gamma values are different between camera devices.
https://github.com/seqsense/robotics-software-sprint-backlog/issues/7#issuecomment-581304117
So I think value validation should not be in this node.

Reply:
If possible, input value is better to be a gamma correction value (1.0 means no-change)

Reply:
It is difficult because of the same reason as the above one.
To realize it, this node requires rosparams to convert gamma correction values to `int` values. It will increase the complexity of this node.

Reply:
Driver nodes are basically for abstracting device interfaces. Generally, it is better to have device-specific parameters in the driver rather than other nodes.

Reply:
Ah... another fatal problem I found is we don't know converting function from `float` to `int`.
(We know that 1.0 is default `int` value, but how about the other values?)

Reply:
How can I deal with it? I want it to be as is.

Reply:
As a driver node perspective, having `gamma_offset` and `gamma_factor` (non-dynamic) parameters would be enough. (if the relation between device parameter integer and actual gamma is a linear function.)

Reply:
Hm, but we don't know correct values of `gamma_offset` and `gamma_factor`, isn't it?

Reply:
At least, we can obtain relative parameter between two devices. Abstracting a difference of the devices is a role of device driver, not one of controller node.

Reply:
All right!